### ISSUE TITLE
Missing reaction / KeyError rebuild_rxn method

### ISSUE THREAD

**Issue by tduigou** [brsynth/rxn_rebuild#1]
Package version: 2.1.2 (pyhd8ed1ab_0)

Executing...
```
rebuild_rxn(
    rxn_rule_id="RR-02-2b23b4fdef31df55-12-F",
    transfo="CMPD_0000000005=CMPD_0000000028+CMPD_0000000022",
    # tmpl_rxn_id="MNXR101763"
)
```

... returns
```
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
/var/folders/_4/1rd7r9d57nl35yy6w9cc_kb40000gn/T/ipykernel_65799/3226665774.py in <module>
      1 from rxn_rebuild import rebuild_rxn
      2 
----> 3 rebuild_rxn(
      4     rxn_rule_id="RR-02-2b23b4fdef31df55-12-F",
      5     transfo="CMPD_0000000005=CMPD_0000000028+CMPD_0000000022",

~/anaconda3/envs/rp2biosensor-dev/lib/python3.9/site-packages/rxn_rebuild/rxn_rebuild.py in rebuild_rxn(rxn_rule_id, transfo, direction, tmpl_rxn_id, cache, logger)
     58                 direction=direction,
     59                 rxn_rule=rxn_rule,
---> 60                 tmpl_rxn=cache.get('template_reactions')[tpl_rxn_id],
     61                 tmpl_rxn_id=tpl_rxn_id,
     62                 compounds=cache.get('cid_strc'),

KeyError: 'MNXR101763'
```

**Comment by breakthewall** [comment]
Issue reproduced, after a couple of tests I think this is not a bug and the behavior is probably due to the fact that `MNXR101763` did not exist in the METANETX database version `3.2` (the version we use for `rr_cache` on which `rxn_rebuild` relies on).

**Comment by breakthewall** [comment]
re-opened

**Comment by breakthewall** [comment]
fixed in 3.0.3
